frontend/src/Components/EventsCard.jsx

```diff
@@ -17,9 +17,10 @@ const EventsCard = (props) => {
     useEffect(getVolunteersList, [reload]);
 
     useEffect(() => {
-        const checkIfVolunteerSignedForEvent = () => {
+        const checkIfVolunteerSignedForEvent = async () => {
```

Comment:
is there any reason this function is asynchronous ?

Reply:
This was a leftover typo & is removed.